Gemfile

```diff
@@ -1,22 +1,3 @@
 source 'https://rubygems.org'
 
 gemspec
-
```

Comment:
Ah, good point, I'll see what can be done about moving that to the gemspec definition instead.